From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Thu, 9 Jun 2022 20:59:35 +0200
Subject: [PATCH] {For LLVMLinux} drivers/soc/rockchip/dtpm: silence
 -Wgnu-designator

clang on x86-64 reports:

    drivers/soc/rockchip/dtpm.c:15:5: error: use of GNU 'missing ='
    extension in designator [-Werror,-Wgnu-designator]
            [0]{ .name = "rk3399",
               ^
               =
---
 drivers/soc/rockchip/dtpm.c | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/drivers/soc/rockchip/dtpm.c b/drivers/soc/rockchip/dtpm.c
index 5a23784b5221..6acbe99a897b 100644
--- a/drivers/soc/rockchip/dtpm.c
+++ b/drivers/soc/rockchip/dtpm.c
@@ -12,33 +12,33 @@
 #include <linux/platform_device.h>
 
 static struct dtpm_node __initdata rk3399_hierarchy[] = {
-	[0]{ .name = "rk3399",
+	[0] = { .name = "rk3399",
 	     .type = DTPM_NODE_VIRTUAL },
-	[1]{ .name = "package",
+	[1] = { .name = "package",
 	     .type = DTPM_NODE_VIRTUAL,
 	     .parent = &rk3399_hierarchy[0] },
-	[2]{ .name = "/cpus/cpu@0",
+	[2] = { .name = "/cpus/cpu@0",
 	     .type = DTPM_NODE_DT,
 	     .parent = &rk3399_hierarchy[1] },
-	[3]{ .name = "/cpus/cpu@1",
+	[3] = { .name = "/cpus/cpu@1",
 	     .type = DTPM_NODE_DT,
 	     .parent = &rk3399_hierarchy[1] },
-	[4]{ .name = "/cpus/cpu@2",
+	[4] = { .name = "/cpus/cpu@2",
 	     .type = DTPM_NODE_DT,
 	     .parent = &rk3399_hierarchy[1] },
-	[5]{ .name = "/cpus/cpu@3",
+	[5] = { .name = "/cpus/cpu@3",
 	     .type = DTPM_NODE_DT,
 	     .parent = &rk3399_hierarchy[1] },
-	[6]{ .name = "/cpus/cpu@100",
+	[6] = { .name = "/cpus/cpu@100",
 	     .type = DTPM_NODE_DT,
 	     .parent = &rk3399_hierarchy[1] },
-	[7]{ .name = "/cpus/cpu@101",
+	[7] = { .name = "/cpus/cpu@101",
 	     .type = DTPM_NODE_DT,
 	     .parent = &rk3399_hierarchy[1] },
-	[8]{ .name = "/gpu@ff9a0000",
+	[8] = { .name = "/gpu@ff9a0000",
 	     .type = DTPM_NODE_DT,
 	     .parent = &rk3399_hierarchy[1] },
-	[9]{ /* sentinel */ }
+	[9] = { /* sentinel */ }
 };
 
 static struct of_device_id __initdata rockchip_dtpm_match_table[] = {
-- 
